### PR TITLE
Add documentation for Payflow Link payment processor

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds documentation for the PayPal Payflow Link payment processor

## Affected DevDocs pages

- [PayPal Payflow Link](https://devdocs.magento.com/guides/v2.3/graphql/payment-methods/payflow-link.html)
- [getPayflowLinkToken query](https://devdocs.magento.com/guides/v2.3/graphql/reference/paypal-get-payflow-link-token.html)

whatsnew
Added the topics [PayPal Payflow Link](https://devdocs.magento.com/guides/v2.3/graphql/payment-methods/payflow-link.html) and [getPayflowLinkToken query](https://devdocs.magento.com/guides/v2.3/graphql/reference/paypal-get-payflow-link-token.html).

### CODE REVIEW

File: guides/v2.3/graphql/reference/paypal-create-payflow-pro-token.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        }
+    }
+}
+```
+
+## Input attributes
+
+### PayflowLinkTokenInput {#PayflowLinkTokenInput}
+
+The `PayflowLinkTokenInput` object defines the attributes required to receive a Payflow Link  token from PayPal.
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/paypal-create-payflow-pro-token.md]:
```suggestion
The `PayflowLinkTokenInput` object defines the attributes required to receive a Payflow Link token from PayPal.
```

File: guides/v2.3/graphql/reference/paypal-create-payflow-pro-token.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cart_id` | String! | The unique ID that identifies the customer's cart
+
+## Output attributes
+
+### PayflowLinkToken
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/paypal-create-payflow-pro-token.md]:
```suggestion
### PaypalExpressToken
```
?

File: guides/v2.3/graphql/reference/paypal-get-payflow-link-token.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        }
+    }
+}
+```
+
+## Input attributes
+
+### PayflowLinkTokenInput {#PayflowLinkTokenInput}
+
+The `PayflowLinkTokenInput` object defines the attributes required to receive a Payflow Link  token from PayPal.
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/paypal-get-payflow-link-token.md]:
```suggestion
The `PayflowLinkTokenInput` object defines the attributes required to receive a Payflow Link token from PayPal.
```

File: guides/v2.3/graphql/reference/paypal-get-payflow-link-token.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cart_id` | String! | The unique ID that identifies the customer's cart
+
+## Output attributes
+
+### PayflowLinkToken
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/paypal-get-payflow-link-token.md]:
```suggestion
### PaypalExpressToken
```
?

File: guides/v2.3/graphql/reference/paypal-create-payflow-pro-token.md (hunk 2 above)
Comment (keharper) [reply guides/v2.3/graphql/reference/paypal-create-payflow-pro-token.md]: Actually, the heading is correct, and the text is wrong.

File: guides/v2.3/graphql/reference/paypal-create-payflow-pro-token.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+The `PaypalExpressToken` object contains a token returned by PayPal and a set of URLs that allow the buyer to authorize payment and adjust checkout details.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/paypal-create-payflow-pro-token.md]:
```suggestion
The `PayflowLinkToken` object contains a token returned by PayPal and a set of URLs that allow the buyer to authorize payment and adjust checkout details.
```

File: guides/v2.3/graphql/reference/paypal-get-payflow-link-token.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+The `PaypalExpressToken` object contains a token returned by PayPal and a set of URLs that allow the buyer to authorize payment and adjust checkout details.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/paypal-get-payflow-link-token.md]:
```suggestion
The `PayflowLinkToken` object contains a token returned by PayPal and a set of URLs that allow the buyer to authorize payment and adjust checkout details.
```

File: guides/v2.3/graphql/payment-methods/payflow-link.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
+---
+group: graphql
+title: Paypal Payflow Link payment method
```
Comment (keharper) [inline guides/v2.3/graphql/payment-methods/payflow-link.md]:
```suggestion
title: PayPal Payflow Link payment method
```

**Pull request merged**